rollouts/controllers/remotesync_controller.go

```diff
@@ -46,18 +46,35 @@ import (
 )
 
 var (
+	// The RootSync object always gets put in the config-management-system namespace,
```

Comment:
Can we confirm if this isn't a requirement ? (Shouldn't be but having a capability to apply rootsyncs also to specific namespaces allows for multi-tenancy use-cases).

Reply:
It must be in the `config-management-namespace`. Here is the error message I get when trying to apply rootsyncs anywhere else:

```
status:
  conditions:
  - errorSummary:
      errorCountAfterTruncation: 1
      totalCount: 1
    lastTransitionTime: "2023-03-06T20:10:31Z"
    lastUpdateTime: "2023-03-06T20:10:31Z"
    message: RootSync objects are only allowed in the config-management-system namespace,
      not in default
    reason: Validation
    status: "True"
    type: Stalled
```

Reply:
ok. Thanks for confirming it.